HyperPlay-Gaming/hyperplay-desktop-client
# [Fix] Native sideload failing because of relative path handler

Fixed an issue Jacob found when sideloading anything on linux that uses a relative path to the root: for instance: 
`usr/bin/steam` .
Hyperplay fails to launch it because it doesn't get the correct exe path.

### How to test

- This PR is simple but touches the launch command directly, so everything related with lanching should be tested.
- Should be tested on all three platforms
- Test sideloaded games and hyperplay games to see if they will launch fine.
- On Linux test passing things like `usr/bin/steam` or any other software installed on system level as well. 

---

Use the following Checklist if you have changed something on the Backend or Frontend:
